package.json

```diff
@@ -64,11 +65,12 @@
     "@babel/preset-react": "^7.26.3",
     "@babel/preset-typescript": "^7.27.0",
     "@babel/register": "^7.25.9",
+    "@mui/internal-babel-plugin-resolve-imports": "2.0.0",
     "@mui/internal-docs-utils": "^2.0.1",
     "@mui/internal-markdown": "^2.0.4",
     "@mui/internal-scripts": "^2.0.7",
     "@mui/internal-test-utils": "^2.0.7",
-    "@mui/monorepo": "github:mui/material-ui#v7.0.2",
+    "@mui/monorepo": "github:mui/material-ui#259b71c8847a5a0a08976fd501a7207b7d5e8fb1",
```

Comment:
Haven't these scripts been released yet?

Reply:
Not yet. Might be released this week.

Reply:
It's been released. Reverted.